website/www/site/layouts/index.html

```diff
@@ -63,73 +61,105 @@ <h2 class="pillars-title">
 {{ end }}
 
 {{ define "graphic-section" }}
-  <div class="graphic section">
-    <div class="graphic__image">
-      <img src="/images/beam_architecture.png" alt="Beam architecture">
+  <div class="graphic">
+    <div class="quotes">
+      <h2 class="margin">{{ T "home-model-title1" }}</h2>
+      <div class="margin section">
+        <div class="row images desktop">
+          <img src="/images/model-sources-icon.svg" class='icon'/>
+          <img src="/images/model-ellipse.svg"/>
+          <img src="/images/model-processing-icon.svg" class='icon'/>
+          <img src="/images/model-ellipse.svg"/>
+          <img src="/images/model-sinks-icon.svg" class='icon'/>
+        </div>
```

Comment:
I'm a little perplexed by this section, it just looks like this:
![image](https://user-images.githubusercontent.com/675055/106538378-02915500-64b1-11eb-9da0-a75dbc1237c2.png)

Is there supposed to be something more here?

Reply:
Talked to @griscz who told me there's supposed to be some text under each item, like this: 
![image](https://user-images.githubusercontent.com/675055/106539976-dc20e900-64b3-11eb-8fde-4043aae95e46.png)

Also the title should be "How Does It Work?" not "How it works?"

Reply:
Yeah, sorry about this, I missed 1 yaml file, now it looks as it should

Reply:
It looks like the title is still "How it works?" and needs to be changed to "How Does it Work?".